DEVELOPERS.rst

```diff
@@ -48,3 +48,32 @@ understanding of Docker behavior and commands.
     docker run -d -p 11011:11011 -p 11015:11015 caskdata/cdap-sandbox
 
 You now have a new Docker container running with CDAP Sandbox.
+
+Build and Run CDAP Sandbox Locally with Maven
+=============================================
+
+The following builds and runs the CDAP sandbox via mvn package. For more details on development
+environments and build options, please see the
+`BUILD.rst <https://github.com/caskdata/cdap/blob/develop/BUILD.rst>`__ file.
+
+- Obtain a fresh copy of the CDAP (GitHub) repo::
+
+    git clone git@github.com:cdapio/cdap.git
+
+- Build CDAP Sandbox distribution ZIP via mvn::
```

Comment:
It would be good add a section that describes how to include the plugins in the zip file.

Reply:
Good idea! I just added this in the new commit; PTAL.